CollapseLauncher/Classes/GameManagement/GameSettings/StarRail/RegistryClass/Model.cs

```diff
@@ -94,10 +103,10 @@ internal class Model : IGameSettingsValue<Model>
 
         /// <summary>
         /// This defines "<c>Character Quality</c>" combobox In-game settings. <br/>
-        /// Options: Low (2), Medium(3), High(4)
+        /// Options: Low (1), Medium(3), High(4)
```

Comment:
As per comment above, the value for the ``Low`` should be ``2``

Reply:
fixed